apps/public-docsite-v9/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Fluent UI React v9 documentation",
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27363&commitId=99f939d46a64ff661bb3ce2588c556cbc876489c&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Focusable.chromium.png|0| Added|

</details>


<div id="vrtCommentFluentUI-v0"/>